charts/pokt-portal-v2/templates/deployment.yaml

```diff
@@ -73,19 +73,24 @@ spec:
             successThreshold: {{ .Values.middleware.startupProbe.successThreshold | default 1 }}
           {{- end }}
           volumeMounts:
-          {{- if .Values.externalSecrets.enabled }}
-          - mountPath: "/config/"
-            name: external-secrets
-            readOnly: true
-          {{- else }}
-          - mountPath: /config
+          - mountPath: {{ .Values.middleware.configMountPath | default "/config" }}
             name: config
-          {{- end }}
           env:
             - name: CONFIG_FILE
-              value: "/config/config.json"
+              value: {{ printf "%s/%s" ( trimSuffix "/" .Values.middleware.configMountPath ) "config.json"}}
             - name: PORT
-              value: "{{ .Values.service.port | default "8080" }}"
+              value: "{{ .Values.middleware.containerPort | default 80 }}"
+            - name: GATEWAY_PRIVATE_KEY
+              value: {{ .Values.middleware.gatewayPrivateKey | quote }}
```

Comment:
We should probably populate the env vars with a loop instead of hard coding the values

Reply:
I agree, I just wanted to leave this for a further enchancement because I am making a task to review these with backend. I think many of them should be removed.